src/features/offer/components/OfferVenueBlock/useVenueBlock.native.test.ts

```diff
@@ -83,30 +89,63 @@ describe('useVenueBlock with offer address', () => {
     const { result } = renderHook(() =>
       useVenueBlock({
         venue: offerWithAddress.venue,
-        address: offerWithAddress.address,
+        offerAddress: offerWithAddress.address,
       })
     )
 
     expect(result.current.venueName).toBe('PATHE MONTPARNASSE')
   })
 
-  it('should return address from metadata Location', () => {
+  it('should return address from  offer address', () => {
     const { result } = renderHook(() =>
       useVenueBlock({
         venue: offerWithAddress.venue,
-        address: offerWithAddress.address,
+        offerAddress: offerWithAddress.address,
       })
     )
 
     expect(result.current.venueAddress).toBe('75013 PARIS 13, 1 RUE DES CAFÉS')
   })
 
+  it('should return isOfferAddressDifferent to true if offer address different than venue address', () => {
+    const { result } = renderHook(() =>
+      useVenueBlock({
+        venue: offerWithAddress.venue,
+        offerAddress: offerWithAddress.address,
+      })
+    )
+
+    expect(result.current.isOfferAddressDifferent).toEqual(true)
+  })
+
+  it('should return isOfferAddressDifferent to false if offer address different and venue address are the same', () => {
```

Comment:
should return isOfferAddressDifferent to false if offer and venue address are the same ?

Reply:
J'avoue je me suis pas relu, je corrige ^^